script.js

```diff
@@ -0,0 +1,26 @@
+CB.CloudApp.init("lfrsdnhmwmef", "409ca5bb-9dd7-40c9-a578-799793fc03b7");
+const allStores = [];
+const productManager = new CloudObjectManager()
+var query = new CB.CloudQuery("Store");
+query.find({
+	success: function(list){
+		for(let store of list){
+		 const newStore = new Store(store);
```

Comment:
Here you can call `newStore.fetchProducts()`. You can pass it a callback function so when it's done fetching products, it can update the UI.